core/src/main/java/io/questdb/cairo/TxReader.java

```diff
@@ -464,7 +490,7 @@ protected void switchRecord(int readBaseOffset, long readRecordSize) {
     }
 
     protected long unsafeGetRawMemorySize() {
```

Comment:
nit: not to this line but you can remove `unsafeGetRawMemorySize` you added some time ago now, it's not used anymore